config/stp.py

```diff
@@ -512,6 +827,7 @@ def stp_vlan_disable(_db, vid):
     db.mod_entry('STP_VLAN', vlan_name, {'enabled': 'false'})
 
 
+# config spanning_tree vlan forward_delay <vlan-id> <4-30 seconds>
```

Comment:
This command is valid only for PVST. Please add check.

Reply:
I have addressed this comment, please review